client/src/main/java/io/cloudsoft/winrm4j/client/WinRmClient.java

```diff
@@ -67,7 +67,7 @@ public static Builder builder(String endpoint) {
 
     public static class Builder {
         private static final java.util.Locale DEFAULT_LOCALE = java.util.Locale.US;
-        private static final int DEFAULT_OPERATION_TIMEOUT = 60000;
+        private static final long DEFAULT_OPERATION_TIMEOUT = 60 * 60000;
```

Comment:
The default time out was one minute, but testing revealed it to be actually 10 minutes. Perhaps there's a bug hiding somewhere that will make this default timeout 10 hours.